__tests__/app.articles.test.js

```diff
@@ -216,4 +216,57 @@ describe("/api/articles", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments - returns array of comments for an article - date DESC", () => {
+    test("200: returns array of comment objects", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const { body } = response;
+          expect(body).toHaveLength(11);
+          let dateCheck = new Date("2100-01-01").getTime();
+          body.forEach((comment) => {
+            let epoch = new Date(comment.created_at).getTime();
+            expect(epoch).toBeLessThan(dateCheck);
+            dateCheck = epoch;
```

Comment:
Had to use this epoch hacky method here again because jest-sort is breaking my repo